FinniversKit/Sources/DNA/Color/UIColor+FinniversKit.swift

```diff
@@ -480,14 +480,14 @@ public extension UIColor {
     ///   - g: green (0-255)
     ///   - b: blue (0-255)
     ///   - a: alpla (0-1)
-    convenience init(r: CGFloat, g: CGFloat, b: CGFloat, a: CGFloat = 1.0) { // swiftlint:disable:this identifier_name
+    internal convenience init(r: CGFloat, g: CGFloat, b: CGFloat, a: CGFloat = 1.0) { // swiftlint:disable:this identifier_name
```

Comment:
having it empty is equal to having it internal

Reply:
you could also specify the module you want and then call the corresponding method i think

Reply:
I actually tried specifying the module, but the compiler was not happy 😞